lib/FailedCRRProducer.js

```diff
@@ -0,0 +1,61 @@
+'use strict'; // eslint-disable-line strict
```

Comment:
Since the producer and consumer classes are specific to replication service, I think they'd be better located somewhere in extensions/replication/ (maybe creating an extra failedCRR/ dir there can make sense).